src/js/components/RadioButton/StyledRadioButton.js

```diff
@@ -25,14 +26,13 @@ const StyledRadioButtonContainer = styled.label`
   :hover {
     background-color: ${props =>
       normalizeColor(
-        props.theme.radioButton.hover &&
+        !props.disabled &&
```

Comment:
It seems like we don't even want to get into this block of code if the radio button is disabled -- any way we can utilize the `input:not([disabled])` being used in lines 21 and 22? Just thinking about how to keep it as clean as possible.

Reply:
This block of code with not run if the `disabled` prop is used within the `calendar`